lib/checker.js

```diff
@@ -20,7 +20,7 @@ utils.inherits(Checker, StringChecker);
 /**
  * Loads configuration from JS Object. Activates and configures required rules.
  *
- * @param {Object} config
+ * @param {Object} [config]
```

Comment:
If config is the only datum passed, I'd vote for removing the .apply and calling configure with config.